src/controllers/dashboard/library.js

```diff
@@ -76,7 +76,8 @@ import cardBuilder from '../../components/cardbuilder/cardBuilder';
     function renameVirtualFolder(page, virtualFolder) {
         import('../../components/prompt/prompt').then(({default: prompt}) => {
             prompt({
-                label: globalize.translate('LabelNewName'),
+                title: globalize.translate('LabelNewName'),
+                label: globalize.translate('MessageAreYouSureYouWishToRenameMediaFolder'),
```

Comment:
```suggestion
                label: globalize.translate('LabelNewName'),
                description: globalize.translate('MessageAreYouSureYouWishToRenameMediaFolder'),
```

Reply:
When I look at your screenshot, I don't think it should be a question but a note/warning instead. Something like "Renaming a folder clears all all metadata, proceed with caution".